src/ol/render/canvas/Immediate.js

```diff
@@ -83,6 +84,14 @@ class CanvasImmediateRenderer extends VectorContext {
      */
     this.transform_ = transform;
 
+    /**
+     * @private
+     * @type {number}
+     */
+    this.transformRotation_ = transform
```

Comment:
... and here you apply the same transform, but the type says `number`. If you want the rotation here, you need to extract it from the transform, or assign null initially and note that in the type.

Reply:
That is not the end of the statement.  I check if the transform exists before extracting from it.

Reply:
Oh I see. Sorry, my bad.